util.py

```diff
@@ -301,3 +301,20 @@ def as_generator(self):
 
     def as_list(self):
         return list(self.as_generator())
+
+
+def render_kubeconfig(
+    kubernetes_config_name: CliHint(
+        help='Name of the kubernetes config from which to take the kubeconfig.'
+    ),
+    output_file_path: CliHint(help='Path to render the kubeconfig to.'),
+):
```

Comment:
consider (not mandatory) to add a _brief_ docstring explaining the idea of this.